xarray/tests/test_sparse.py

```diff
@@ -879,3 +864,17 @@ def test_sparse_coords(self):
             dims=["x"],
             coords={"x": COO.from_numpy([1, 2, 3, 4])},
         )
+
+
+def test_chunk():
+    s = sparse.COO.from_numpy(np.array([0, 0, 1, 2]))
+    a = DataArray(s)
+    ac = a.chunk(2)
+    assert ac.chunks == ((2, 2),)
+    assert isinstance(ac.data._meta, sparse.COO)
+    assert_identical(ac, a)
```

Comment:
@shoyer After #3204, ``assert_identical(a, ac)`` does not work anymore - not even with the latest sparse git head

Reply:
I'm guessing this is due to https://github.com/dask/dask/issues/5259